pkg/discovery/monitoring/master/cluster_monitor.go

```diff
@@ -14,17 +14,19 @@ import (
 	"github.com/golang/glog"
 )
 
+// This monitor is based on Kubenetes's Node, Pod and Container settings;
+// it will mainly generate CPU/Memory commodity's Capacity and Provision for Node, Pod, and Container;
 type ClusterMonitor struct {
 	config *ClusterMonitorConfig
 
+	//TODO: since this sink is not accessed by multiple goroutines
```

Comment:
help to confirm that this **sink** is not accessed concurrently, so the synchronized Add() is not necessary.